### ISSUE TITLE
Failed to login

### ISSUE THREAD

**Issue by EysseW** [lordme1iodas/cordless#2]
## How have you installed cordless

- [x] Arch User Repository

- [ ] scoop
- [ ] brew
- [ ] go get
- [ ] git clone & go build .

## What happened

I got an error message on trying to log in. Happens both when I do it with the token as when I do it with email + password

## Error output

strconv.ParseInt: parsing "1709703206.411": invalid syntax 

## How do you reproduce this bug

Install the program and try to log in (and probably more but I wouldn't know)

## System information

| Key | Value |
| - | - |
| OS | Archlinux
| Architecture | amd64
| Terminal | foot

**Comment by wynwxst** [comment]
> ## How have you installed cordless
> * [x]  Arch User Repository
> 
> * [ ]  scoop
> * [ ]  brew
> * [ ]  go get
> * [ ]  git clone & go build .
> 
> ## What happened
> I got an error message on trying to log in. Happens both when I do it with the token as when I do it with email + password
> 
> ## Error output
> strconv.ParseInt: parsing "1709703206.411": invalid syntax
> 
> ## How do you reproduce this bug
> Install the program and try to log in (and probably more but I wouldn't know)
> 
> ## System information
> Key	Value
> OS	Archlinux
> Architecture	amd64
> Terminal	foot

It won't work anymore because of discord's reformatting of tokens sadly. It requires an entire rewrite, if you're up for the task I can explain how to start.

**Comment by EysseW** [comment]
Yes, I'd be glad to do so actually. What language is it written in?

**Comment by wynwxst** [comment]
> Yes, I'd be glad to do so actually. What language is it written in?

It's written in go like go the gopher language. The best way to start would be to pick a language with a discord based api that you're familiar with or if you know go then lucky you! The next thing to do would be to read the discord docs for how exactly the tokens have changed and then embed it within your code
